Microsoft.Toolkit.Uwp.UI/Extensions/Visual/VisualEx.cs

```diff
@@ -0,0 +1,739 @@
+        {
+            if (e.NewValue is string str)
+            {
+                SetAnchorPoint(d, str);
+            }
+        }
+
+        private static void OnCenterPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetCenterPoint(d, e.NewValue as string);
+            }
+        }
+
+        private static void OnOffsetChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetOffset(d, str);
+            }
+        }
+
+        private static void OnOpacityChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetOpacity(d, dbl);
+            }
+        }
+
+        private static void OnRotationAngleChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetRotationAngle(d, dbl);
+            }
+        }
+
+        private static void OnRotationAngleInDegreesChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetRotationAngleInDegrees(d, dbl);
+            }
+        }
+
+        private static void OnRotationAxisChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetRotationAxis(d, str);
+            }
+        }
+
+        private static void OnScaleChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetScale(d, str);
+            }
+        }
+
+        private static void OnSizeChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetSize(d, str);
+            }
+        }
+
+        private static void OnKeepCenterPointCenteredChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (d is FrameworkElement element && !DesignTimeHelpers.IsRunningInLegacyDesignerMode)
+            {
+                SetupKeepCenterPointCentered(e, element);
+            }
+        }
+
+        private static void SetupKeepCenterPointCentered(DependencyPropertyChangedEventArgs e, FrameworkElement element)
+        {
+            element.SizeChanged -= KeepCenteredElementSizeChanged;
+
+            if (e.NewValue is bool keepCentered)
+            {
+                if (keepCentered)
+                {
+                    element.SizeChanged -= KeepCenteredElementSizeChanged;
+                    element.SizeChanged += KeepCenteredElementSizeChanged;
```

Comment:
🤢 +/-
This static eventhandler will forever hold on to the element. No Bueno ;)